src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -241,16 +247,22 @@ private static boolean printStackTraceOnUserExceptions() {
      * Returns the command line program specified, or prints the usage and exits with exit code 1 *
      */
     private static CommandLineProgram extractCommandLineProgram(final String[] args, final List<String> packageList, final List<Class<? extends CommandLineProgram>> classList, final String commandLineName) {
+
         /** Get the set of classes that are our command line programs **/
         final ClassFinder classFinder = new ClassFinder();
         for (final String pkg : packageList) {
+            classFinder.find(pkg, picard.cmdline.CommandLineProgram.class);
             classFinder.find(pkg, CommandLineProgram.class);
         }
         String missingAnnotationClasses = "";
         final Set<Class<?>> toCheck = classFinder.getClasses();
         toCheck.addAll(classList);
         final Map<String, Class<?>> simpleNameToClass = new LinkedHashMap<>();
+        final Set<String> maskedPicardToolClasses = getMaskedToolClasses();
         for (final Class<?> clazz : toCheck) {
+            if (maskedPicardToolClasses.contains(clazz.getName())) {
```

Comment:
Instead of using `clazz.getName()`, here a direct comparison of classes will be better if there is a common interface....